### PR TITLE
Move vsc.install to vsc-install

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,35 +33,21 @@
...
     'name': 'vsc-base',
-    'version': '2.4.2',
+    'version': '2.5.0',
     'author': [sdw, jt, ag, kh],
     'maintainer': [sdw, jt, ag, kh],
-    'packages': ['vsc', 'vsc.install', 'vsc.utils'],
+    'packages': ['vsc.utils'],
     'scripts': ['bin/logdaemon.py', 'bin/startlogdaemon.sh', 'bin/bdist_rpm.sh', 'bin/optcomplete.bash'],
-    'install_requires' : ['setuptools'],
+    'install_requires': ['vsc-install >= 1.0.0'],
```
Comment (boegel) [inline setup.py]: no

**Comment by boegel** [comment]
remove this from `vsc/__init__.py`:

```
# here for backwards compatibility
from vsc.utils import fancylogger
```

File: setup.py (hunk 1 above)
Comment (boegel) [inline setup.py]: or add a comment why (because of the `from vsc.utils import fancylogger` backward compatibilty thingie)

**Comment by stdweird** [comment]
retest this please

**Comment by stdweird** [comment]
retest this please

**Comment by stdweird** [comment]
retest this please

**Comment by stdweird** [comment]
@boegel tests passed

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,9 @@
 *.py[co]
 *.swp
+*~
+
+# ok to ignore generated fiel in vsc-base
```
Comment (boegel) [inline .gitignore]: fiel?

File: setup.py [hunk 3]
```diff
@@ -33,35 +33,21 @@
...
-    'version': '2.4.2',
+    'version': '2.4.9',
     'author': [sdw, jt, ag, kh],
     'maintainer': [sdw, jt, ag, kh],
-    'packages': ['vsc', 'vsc.install', 'vsc.utils'],
+    'packages': ['vsc.utils'],
     'scripts': ['bin/logdaemon.py', 'bin/startlogdaemon.sh', 'bin/bdist_rpm.sh', 'bin/optcomplete.bash'],
-    'install_requires' : ['setuptools'],
+    'install_requires': ['vsc-install >= 0.9.0'], # as long as 1.0.0 is not out, vsc-base should still provide vsc.fancylogger
+    'setup_requires': ['vsc-install >= 0.9.0'],
```
Comment (boegel) [inline setup.py]: introduce a `VSC_INSTALL_REQ_VERSION` constant?

File: setup.py [hunk 4]
```diff
@@ -33,35 +33,23 @@
...
-def remove_bdist_rpm_source_file():
-    """List of files to remove from the (source) RPM."""
-    return []
-
-shared_setup.remove_extra_bdist_rpm_files = remove_bdist_rpm_source_file
-
 # Re-reload the vsc modules that vsc-base ships
 shared_setup.RELOAD_VSC_MODS = True
 
+VSC_INSTALL_REQ_VERSION = 'vsc-install >= 0.9.0'
```
Comment (boegel) [inline setup.py]: no, make this `0.9.0` here

**Comment by boegel** [comment]
let's see what this breaks... ;-)

**Pull request merged**